### PR TITLE
February Release Roundup

### PR DESCRIPTION
**Pull request by dmarvp**

### CODE REVIEW

File: releases/202002.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Bespoken Release Roundup - February 2020
+We haven't stopped working hard for you in this 2020 and to prove it here's the latest features, improvements and bug fixes that were released during this month.
```
Comment (jkelvie) [inline releases/202002.md]: "in this 2020 and to prove it" - "in 2020! To prove it,"

File: releases/202002.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## [Dashboard](https://apps.bespoken.io)
+### New Features and Improvements
+- **Advanced Test Settings**: By clicking on the gear icon on the test page, you can now adjust the execution of your tests using the configuration described [here](https://read.bespoken.io/end-to-end/guide/#configuration). This includes homophones, timeouts and others. Be aware that key settings like the virtual device, locales and voices will keep using the ones selected on the dropdowns.
```
Comment (jkelvie) [inline releases/202002.md]: "keep" -> "continue"

File: releases/202002.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Bespoken Release Roundup - February 2020
+We haven't stopped working hard for you in this 2020 and to prove it here's the latest features, improvements and bug fixes that were released during this month.
+
+## [Dashboard](https://apps.bespoken.io)
+### New Features and Improvements
+- **Advanced Test Settings**: By clicking on the gear icon on the test page, you can now adjust the execution of your tests using the configuration described [here](https://read.bespoken.io/end-to-end/guide/#configuration). This includes homophones, timeouts and others. Be aware that key settings like the virtual device, locales and voices will keep using the ones selected on the dropdowns. 
+
+![Advanced Settings](./images/202002_01.gif)
+
+- **Twilio support on Dashboard**: Our IVR users will be happy to know that you can now create Virtual Devices for Twilio and create tests for it too. For now, this feature will be activated by request so, if you are interested in creating these kinds of tests just get in touch with us!
```
Comment (jkelvie) [inline releases/202002.md]: "...create tests for it too. Automating testing of telephone systems now with Bespoken."

File: releases/202002.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- **Improved error messaging**: When facing an unknown exception, bst would usually reply with messages saying `Error description missing`, `Cannot read property 'message' of undefined` and so on. We've been working actively to make those messages clearer so that you can know exactly the reason why they happened.
+- **HTML Report on a single file**: Users of unit and end to end testing can now generate HTML reports of their test runs in a single file with styles and scripts embedded in it. This is helpful if you want to share your report with others. To do this, just set the following environment variable: `JEST_STARE_INLINE_SOURCE=true`. After running your tests, search for the `index.inline.html` file in your `test_output/report` folder.
```
Comment (jkelvie) [inline releases/202002.md]: "Single-File HTML Report"

File: releases/202002.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+- **Support for env files**: BST supports the usage of `.env` files now. This is helpful for Jest Stare settings like the one described above and more.
+
+### Bug fixing
+- Fixed a bug in which our card shorthand properties were not being filled correctly for Google actions.
```
Comment (jkelvie) [inline releases/202002.md]: "fillled" -> "populated"

**Review by jkelvie — CHANGES_REQUESTED** [review]

**Review by jkelvie — CHANGES_REQUESTED** [review]
I would also recommend setting the images to 50% height

**Comment by dmarvp** [comment]
> I would also recommend setting the images to 50% height

@jkelvie images will be adjusted when published on the bespoken blog :)

**Review by jkelvie — APPROVED** [review]

**Pull request merged**